Project.toml

```diff
@@ -2,7 +2,7 @@ name = "CodecLz4"
 uuid = "5ba52731-8f18-5e0d-9241-30f10d1ec561"
 license = "MIT"
 author = "Invenia Technical Computing"
-version = "0.2.1"
+version = "0.2.2"
```

Comment:
Would this not be a `0.3` release since we're changing the function parameter types?

Reply:
Apart from the deprecation of `LZ4CompressorStream` and `LZ4CompressorStream`, nothing should change in any existing functions but I suppose this could be a `0.3`.

Reply:
We're removing the export of: [LZ4Compressor, LZ4CompressorStream, LZ4Decompressor, LZ4DecompressorStream].

Wouldn't that break someones code if we released this as `0.2.2` and they auto-upgraded?

Reply:
IIRC `@deprecate_binding` auto-exports the deprecated functions. You can see it in action here https://github.com/invenia/CodecLz4.jl/pull/26/files?file-filters%5B%5D=.jl#diff-8177ddafcfa7afb66b92a2e544a2590aR134